lib/hal/led.h

```diff
@@ -0,0 +1,18 @@
+#ifndef HAL_LED_H
+#define HAL_LED_H
+
+#include <hal/xbox.h>
+#include <xboxkrnl/xboxkrnl.h>
+
+#define LED_OFF    0x00
+#define LED_GREEN  0x01
+#define LED_RED    0x10
+#define LED_AMBER  0x11
+
+#define SMC_REG_LEDMODE 0x07
+#define SMC_REG_LEDSEQ  0x08
```

Comment:
These should be in a different file - for now, I'd probably move them into the C file. We should expose them more generally from xboxkrnl.h or similar in the future (in a more complete set).

Reply:
Agreed. :+1: